types/node/http.d.ts

```diff
@@ -1031,7 +1031,7 @@ declare module "http" {
          * The request method.
          * @since v0.1.97
          */
-        method: string;
+        method: string | null;
```

Comment:
Isn't this defaulted?

https://github.com/nodejs/node/blob/741975041995a272ff7e378bcbb6d6fa4b93f38f/lib/_http_client.js#L220-L233

Reply:
Yes. You're right.